vtexdocs/devportal
# Fix/mermaid

#### What is the purpose of this pull request?

To solve [issues](https://docs.google.com/document/d/1co8IMT4cPf53kdy0aHLWsO1wem0amxJcIfCbJUaid_8/edit#) with the mermaid diagrams.

#### What problem is this solving?

- Inconsistent style
- Now you are not able to zoom using the wheel
- Fixed infinite panning after moving the cursor near the diagram borders

[Further information and instructions on how to add style to diagrams](https://www.notion.so/vtexhandbook/Corre-es-em-diagramas-mermaid-f230601a227f47ff82164e8403b42382).

#### How should this be manually tested?

Open the following pages in the preview and check if the diagrams are displayed as expected:
- [PII data architecture specifications](https://deploy-preview-399--elated-hoover-5c29bf.netlify.app/docs/guides/pii-data-architecture-specifications)
- [Data residency](https://deploy-preview-399--elated-hoover-5c29bf.netlify.app/docs/guides/data-residency)

#### Screenshots or example usage

![image](https://github.com/vtexdocs/devportal/assets/62757720/2870b9d5-bc08-4249-b25c-8bd7487a2c9f)
![image](https://github.com/vtexdocs/devportal/assets/62757720/d95fe34d-35e8-4af2-b503-ffffc2cdb93c)

#### Types of changes
